pkg/builder/tracing_build_queue.go

```diff
@@ -0,0 +1,42 @@
+package builder
```

Comment:
Similar to what I stated in https://github.com/buildbarn/bb-remote-execution/pull/67, I don't think stuff like this should be added. Some generic gRPC middleware could do this for us.